db/schema.go

```diff
@@ -28,8 +28,9 @@ func (s *Schema) ToErd(showIndex bool) string {
 
 	for _, table := range s.Tables {
 		for _, foreignKey := range table.ForeignKeys {
-			if tableNames.Contains(foreignKey.ToTable) {
-				lines = append(lines, fmt.Sprintf("%s }-- %s", table.Name, foreignKey.ToTable))
+			toTable := strings.ToLower(foreignKey.ToTable)
```

Comment:
Please add a test case for this change to `db/schema_test.go` if possible 🙏

Reply:
Unfortunately, I am not fully familiar with the tests written, so i am not able to enhance them.

Reply:
OK, I'll write later.

Reply:
I add tests at #227